apps/api/src/coordinator/coordinator.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Post, Body, Patch, Param, Delete } from '@nestjs/common'
+import { Public } from 'nest-keycloak-connect'
+import { CoordinatorService } from './coordinator.service'
+import { CreateCoordinatorDto } from './dto/create-coordinator.dto'
+import { UpdateCoordinatorDto } from './dto/update-coordinator.dto'
+
+@Controller('coordinator')
+export class CoordinatorController {
+  constructor(private readonly coordinatorService: CoordinatorService) {}
+
+  @Post()
+  @Public()
+  create(@Body() createCoordinatorDto: CreateCoordinatorDto) {
+    return this.coordinatorService.create(createCoordinatorDto)
+  }
+
+  @Get('list')
+  @Public()
+  findAll() {
+    return this.coordinatorService.findAll()
+  }
+
+  @Get(':id')
+  @Public()
+  findOne(@Param('id') id: string) {
+    return this.coordinatorService.findOne(id)
+  }
+
+  @Patch(':id')
+  @Public()
+  update(@Param('id') id: string, @Body() updateCoordinatorDto: UpdateCoordinatorDto) {
+    return this.coordinatorService.update(id, updateCoordinatorDto)
+  }
```

Comment:
PATCH method should not be allowed for a Coordinator, since there are no fields that can be changed in him. The actual changes are in the Person model so if Person names need to be changed that should happen through the Person PATCH api endpoint